azure-pipelines-integration.yml

```diff
@@ -0,0 +1,25 @@
+# Branches that trigger a build on commit
+trigger:
+- master
+
+# Branches that trigger builds on PR
+pr:
+- master
+
+jobs:
+
+- job: Windows
+  pool: dotnet-external-temp
```

Comment:
I don't think we can use this pool anymore, it should be
```yaml
  pool:
    name: NetCorePublic-Int-Pool
    queue: buildpool.windows.10.amd64.vs2017.open
```

Reply:
You can also run this on the big queue (the default one for AzDO) if we don't have any special VM dependencies  
```yaml
  pool:
    vmImage: 'vs2017-win2016'
```